src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -1,5 +1,64 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
The schema comment thing was bulk removed by a tool.  Please leave it removed so it doesn't just flip-flop as gone and back.

Reply:
@danmosemsft had removed them explicitly.  At the time I'd commented that VS would just add them back and thus I didn't like removing them, as I didn't want to fight with VS.  I actually prefer just letting VS do its thing here.

Reply:
If we have precedence one way or the other I'm happy to go with it.  I edit the resx files with vim, so the rest of crypto probably doesn't have them.  But I'll admit to having a limited view here.